java-client/src/main/java/org/opensearch/client/opensearch/OpenSearchClient.java

```diff
@@ -155,6 +156,9 @@ public OpenSearchClient withTransportOptions(@Nullable TransportOptions transpor
     }
 
     // ----- Child clients
+    public OpenSearchGenericClient generic() {
+        return new OpenSearchGenericClient(this.transport, this.transportOptions);
+    }
```

Comment:
Why do we need a new kind of client? Why can't I use the existing client?

Reply:
Existing clients are all about object model: requests and responses are all typed. The idea of generic one is basically allow to send raw requests and receive raw response (by raw I mean just as bytes).